packages/vue/src/components/molecules/SfAlert/SfAlert.js

```diff
@@ -13,38 +13,29 @@ export default {
       type: String,
       default: ""
     },
-    /** One of predefined SfIcon colors, default is black.  */
-    colorIcon: {
-      type: String,
-      default: "white"
-    },
-    /** One of predefined SfIcon sizes.  */
-    sizeIcon: {
-      type: String,
-      default: "24px"
-    },
-    /** Svg file iconPath  */
-    icon: {
-      type: String,
-      default: "clock"
-    },
     /**
-     * Alert type ("alert", "warning", "info", "success"). Check "Knobs" section to see how they look like.
+     * Alert type ("secondary", "info", "success", "warning", "danger"). Check "Knobs" section to see how they look like.
      */
     type: {
       type: String,
-      default: "alert",
+      default: "secondary",
```

Comment:
i think we shouldn't propose `secondary` as an option here, it's available through utility classes but doesn't fit the alert's context, wdyt?

Reply:
agreeing with @filrak's comment, imo the default should be `info`